WEBROOOOT/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	
+	<link rel="stylesheet" href="style.css">
+
+	 <title>ABOUT ME</title>
+
+
+	 <nav> 
```

Comment:
The <nav> tag should go inside the <body> tag.  Tags in the <head> tag are really reserved for the "behind the scenes" parts of the page - i.e. linking to stylesheets, page title (that appears in your browser's tab).

Otherwise, this is a great use of the <nav> tag.